nginx/Dockerfile

```diff
@@ -13,14 +13,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM quay.io/kubernetes-ingress-controller/nginx-ingress-controller:0.25.0
+FROM quay.io/kubernetes-ingress-controller/nginx-ingress-controller:0.27.1
```

Comment:
https://quay.io/repository/kubernetes-ingress-controller/nginx-ingress-controller?tag=latest&tab=tags

Reply:
requires update to helm chart:
https://github.com/kubernetes/ingress-nginx/blob/master/Changelog.md
runAsUser: 101

Reply:
Change included in this PR https://github.com/astronomer/helm.astronomer.io/pull/315 and the existing helm chart CI will automatically cover this failure scenario